bootstrap-extensions/pom.xml

```diff
@@ -49,6 +49,23 @@
         </dependency>
 
         <dependency>
+            <groupId>de.agilecoders.wicket.webjars</groupId>
+            <artifactId>wicket-webjars</artifactId>
```

Comment:
This should already be in the classpath as transitive dependency from bootstrap-core.